app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/EditEvent.kt

```diff
@@ -0,0 +1,311 @@
+          }
+        },
+        bottomBar = {
+          BottomNavigationMenu(
+              onTabSelect = { selectedTab ->
+                nav.navigateTo(TOP_LEVEL_DESTINATIONS.first { it.route == selectedTab })
+              },
+              tabList = TOP_LEVEL_DESTINATIONS,
+              selectedItem = Route.EVENTS)
+        },
+        content = { innerPadding ->
+          Box(modifier = Modifier.padding(innerPadding)) {
+            Column(
+                modifier =
+                    Modifier.padding(start = 15.dp, end = 15.dp)
+                        .verticalScroll(rememberScrollState(0))
+                        .fillMaxSize(),
+                verticalArrangement = Arrangement.Top,
+                horizontalAlignment = Alignment.CenterHorizontally) {
+                  Image(
+                      painter =
+                          if (imageBitmap != null) {
+                            androidx.compose.ui.graphics.painter.BitmapPainter(imageBitmap!!)
+                          } else if (!profilePictureUrl.isNullOrEmpty()) {
+                            rememberAsyncImagePainter(profilePictureUrl)
+                          } else {
+                            painterResource(id = R.drawable.gomeet_logo)
+                          },
+                      contentDescription = "Event picture",
+                      modifier =
+                          Modifier.padding(start = 15.dp, end = 15.dp, top = 30.dp, bottom = 15.dp)
+                              .width(101.dp)
+                              .height(101.dp)
+                              .clickable { imagePickerLauncher.launch("image/*") }
+                              .clip(CircleShape)
+                              .background(color = MaterialTheme.colorScheme.background)
+                              .align(Alignment.CenterHorizontally)
+                              .testTag("Event Picture"),
+                      contentScale = ContentScale.Crop)
+
+                  Spacer(modifier = Modifier.size(16.dp))
+
+                  TextField(
+                      value = titleState.value,
+                      onValueChange = { newValue -> titleState.value = newValue },
+                      label = { Text("Title") },
+                      singleLine = true,
+                      modifier = Modifier.fillMaxWidth(),
+                      colors = textFieldColors)
+                  Spacer(modifier = Modifier.size(16.dp))
+
+                  TextField(
+                      value = descriptionState.value,
+                      onValueChange = { newValue -> descriptionState.value = newValue },
+                      label = { Text("Description") },
+                      singleLine = true,
+                      modifier = Modifier.fillMaxWidth(),
+                      colors = textFieldColors)
+                  Spacer(modifier = Modifier.size(16.dp))
+
+                  LocationField(selectedLocation, locationState, eventViewModel)
```

Comment:
The location text field doesn’t have the same width and style as the other text fields.

The link text field doesn't have the same style as well

<img width="281" alt="Capture d’écran 2024-05-21 à 17 42 43" src="https://github.com/SwEnt-Project-G18/SwEntApp/assets/92448776/a33c0740-99f7-45ef-bc16-73e39d779367">

Reply:
Good catch, that should be fixed now.